vespa_analysis/vespa_msa.py

```diff
@@ -0,0 +1,67 @@
+import os
+import argparse
+from Bio import SeqIO
+import pandas as pd
+import numpy as np
+
+
+def sav_stats(gen_id,gen_aln,vespa_map_out):
+    sav_vector = []
+    c = 0
+    df = pd.read_csv(vespa_map_out)
+    seq = gen_aln
+
+    for j in seq:
+        if j == '_':
+            sav_vector.append(0)
+        else:
+            c=c+1
+            if c-1 in list(df['Gen Seq Index'].values):
+                sav_vector.append(list(df['Score'].values)[list(df['Gen Seq Index'].values).index(c-1)]*10)
+            else:
+                sav_vector.append(0)
+    return sav_vector
+
+def cons_stats(ref_id,ref_aln,cons_class):
+    cons_class = cons_class.split(',')
+    ref_seq = ref_aln
+    cons_vector = []
+    c = 0
+    for j in ref_seq:
+        if j == '_':
+            cons_vector.append(0)
+        else:
+            c=c+1
+            cons_vector.append(int(cons_class[c-1]))
+    return cons_vector
+
+parser = argparse.ArgumentParser(description='Computes SAV and Conservation scores statistics')
```

Comment:
we can group all of these arguments into one function and call it at the beginning of the script to parse all of these arguments.